recipes/SignalTrisJs/src/Main.purs

```diff
@@ -398,11 +397,6 @@ setFillStyleC :: Context2D -> Color -> Effect Unit
 setFillStyleC ctx = setFillStyle ctx <<< toHexString
 
 -----------------
--- Faster versions of these functions are proposed in
--- https://github.com/purescript/purescript-ordered-collections/pull/21
-toMap :: forall a. Ord a => Set a -> Map a Unit
-toMap = foldl (\m k -> Map.insert k unit m) mempty
-
 annotateSet :: forall a b. Ord a => (a -> b) -> Set a -> Map a b
 annotateSet f = mapWithIndex (\k _ -> f k) <<< toMap
```

Comment:
[`toMap` is now in `Data.Set`](https://github.com/purescript/purescript-ordered-collections/pull/31/files), so no need to implement `Monoid`. I couldn't find anything like `annotateSet` in there though.

Reply:
Just added `annotateSet` to the todo list in https://github.com/purescript/purescript-ordered-collections/issues/50

Reply:
1616b74735f1a6bc2d1950f065e366ac6bca71c9